### DISCUSSION TITLE
Unable to authenticate: Unable to log in...

### DISCUSSION THREAD

**Discussion by neiomi1** [felddy/foundryvtt-docker#874] [Q&A] [answered]
When trying to start the container with my Credentials in the environment, I suddenly get stuck in this loop. I checked the credentials and they work fine on the website (Didn't change them either but after restarting the container today it didn't work anymore :/ )
![grafik](https://github.com/felddy/foundryvtt-docker/assets/48922182/57321086-fd96-4865-bf92-986c94bd8fb9)

Using a timed link worked
![grafik](https://github.com/felddy/foundryvtt-docker/assets/48922182/c52f5cdc-6225-4c4e-9415-f03966925573)

Did something change in how verification is handled?

**Comment by felddy** [comment, marked as answer]
Pull the new release: [`11.315.1`](https://github.com/felddy/foundryvtt-docker/releases/tag/v11.315.1)

See:
- https://github.com/felddy/foundryvtt-docker/discussions/867
